### PR TITLE
[DO NOT MERGE] dev-libs/sway: 0.15.1 version bump

### PR DESCRIPTION
**Pull request by Hummer12007**
changed branch in live, dropped old versions.

0.15.2 is coming really soon.

### CODE REVIEW

File: dev-libs/sway/sway-0.15.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+DESCRIPTION="i3-compatible Wayland window manager"
+HOMEPAGE="http://swaywm.org/"
+
+SRC_URI="https://github.com/swaywm/sway/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+swaybg +swaybar +swaymsg swaygrab swaylock +gdk-pixbuf zsh-completion wallpapers systemd +tray"
```
Comment (titanofold) [inline dev-libs/sway/sway-0.15.1.ebuild]: Alphabetize.

File: dev-libs/sway/sway-0.15.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+REQUIRED_USE="tray? ( swaybar )"
+
+RDEPEND=">=dev-libs/wlc-0.0.8[systemd=]
```
Comment (titanofold) [inline dev-libs/sway/sway-0.15.1.ebuild]: Alphabetize, grouping use conditionals, version requirements, and bare dependencies.

File: dev-libs/sway/sway-0.15.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	x11-libs/libxkbcommon
+	dev-libs/wayland
+	sys-libs/libcap
+	x11-libs/pango
+	x11-libs/cairo
+	swaylock? ( virtual/pam )
+	tray? ( sys-apps/dbus )
+	gdk-pixbuf? ( x11-libs/gdk-pixbuf[jpeg] )"
+
+DEPEND="${RDEPEND}
```
Comment (titanofold) [inline dev-libs/sway/sway-0.15.1.ebuild]: Alphabetize.

**Comment by Hummer12007** [comment]
Fixed

**Comment by monsieurp** [comment]
@Hummer12007 @titanofold can we merge this PR?

**Comment by Hummer12007** [comment]
@monsieurp ah, ok

**Comment by monsieurp** [comment]
@Hummer12007 ?? this is a simple yes/no question.

**Comment by Hummer12007** [comment]
Yes.

**Comment by embik** [comment]
Please merge this, sway-0.15.0 doesn't build against the current `dev-libs/json-c` version anymore.

**Pull request closed without merging**